docs/setup/mcp-setup.md

```
@@ -353,16 +353,20 @@ Test that everything is working:
 Once setup is complete, see [Tools Overview](../guides/mcp-tools-overview.md) for detailed information about available capabilities and usage examples.
 
-## Troubleshooting
+## Verify Your Setup
 
-If you encounter issues after setup, use the built-in diagnostics:
+After completing setup, verify everything is working correctly:
 
-```bash
-# In your MCP client, ask:
-"Show dot-ai status"
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)